components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/dao/sql/HumanTaskJPQLQueryBuilder.java

```diff
@@ -256,7 +256,7 @@ private Query buildAssignedToMeQuery() {
         }
         Query assignedToMeQuery = em.createQuery(SELECT_DISTINCT_TASKS +
                 JOIN_HUMAN_ROLES_JOIN_ORG_ENTITIES +
-                " oe.name = :name " +
+                "LOWER(oe.name) = :name " +
```

Comment:
Can we check LOWER(oe.name) = LOWER(:name). Then we can avoid havinf to pass the lowercase name in the query parameter.

Reply:
Fixed